roles/icinga_agent/tasks/main.yml

```diff
@@ -66,6 +77,14 @@
     - icinga_agent_enable_features | length > 0
 
 - block:
+    - name: create certs folder in case it is missing
+      ansible.builtin.file:
+        path: /var/lib/icinga2/certs/ca.crt
```

Comment:
Is this correct? A directory called ca.crt?

Reply:
No this is not correct. I fixed it right away.